ecrtokenrefresher/pkg/secrets/registrymirror/registrymirror.go

```diff
@@ -0,0 +1,88 @@
+package registrymirror
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/aws/eks-anywhere-packages/ecrtokenrefresher/pkg/constants"
```

Comment:
Import statement order should be fix, suprised this wasn't caught by golang-ci

Reply:
I ran golang-ci locally but didn't get complaints about that. I also ran gofmt to format all code.